ic-os/components/hostos-scripts/generate-guestos-config/dev-generate-guestos-config.sh

```diff
@@ -83,7 +83,6 @@ function read_variables() {
 function assemble_config_media() {
     cmd=(/opt/ic/bin/build-bootstrap-config-image.sh ${MEDIA})
     cmd+=(--nns_public_key "/boot/config/nns_public_key.pem")
-    cmd+=(--elasticsearch_hosts "$(/opt/ic/bin/fetch-property.sh --key=.logging.hosts --metric=hostos_logging_hosts --config=${DEPLOYMENT})")
```

Comment:
I've removed the elasticsearch_hosts flag from both generate-guestos-config files because fetch-property fails if the value is empty in deployment.json.

The field is still in build-bootstrap-config-image.sh, which testing uses directly.

Note that once we are on the new config, the elasticsearch_hosts field will be contained directly in the config object.

Reply:
This _will_ break the configuration for testnets, though, right?

I don't think any tests that use HostOS directly are wired up for logs, so that's okay, but I'm pretty sure this same script is used along the systest path.

Are those logging targets injected, or were they relying on these defaults?

Reply:
Good question. From what I can tell, the static testnets are injecting the elasticsearch_hosts from this [shared-config.yml](https://github.com/dfinity/ic/blob/83d9514ef8e15c20758171c692f58361cd92a055/testnet/env/shared-config.yml#L70) file. 

But they also don't use generate-guestos-config, they use [build-bootstrap-config-image](https://github.com/dfinity/ic/blob/e647d84f466364b5c2494c50d3a6438c0d5f452d/testnet/tools/build-guestos-configs.sh#L319) directly, which is unchanged, so this change shouldn't have any effect on static testnets.

Reply:
Sorry, I'm always forgetting that this is still two scripts.

I was thinking about farm testnets, but these also use `build-bootstrap-config-image`, so nothing to worry about.